test/acceptance_tests/acceptancetest_multiple_zones_display.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+import random
+
+import rospy
+from std_msgs.msg import Byte
+
+if __name__ == "__main__":
+    rospy.init_node("multiple_zones_test")
+
+    pub = rospy.Publisher("/relay_cmd", Byte, queue_size=10)
+
+    r1 = rospy.Rate(1)  # 1hz
+    r10 = rospy.Rate(10)  # 10hz
+    while not rospy.is_shutdown():
+        pub.publish(0)
+        r10.sleep()
+        pub.publish(8)
+        r10.sleep()
+        pub.publish(0)
+        if random.random() > .5:
+            r10.sleep()
+            pub.publish(8)
```

Comment:
would make it easier to understand what is happening if you would extract the relay_cmd byte and sleeps.

```suggestion
        switch_to_zone(zone_id=0)
        switch_to_zone(zone_id=1)
        switch_to_zone(zone_id=0)
        if random.random() > .5:
            switch_to_zone(zone_id=1)
```